content/2017/09/19/susy-use.rst

```diff
@@ -0,0 +1,175 @@
+public: yes
+tags: [Susy, Susy3, CSS, Sass, Grids, Code]
+author: miriam
+image:
+  - src: 'projects/susy.png'
+headline:
+  - type: 'Code Snippets'
+    tagline: 'Practical media helpers for Susy3'
+summary: |
+  **Most grids change with the viewport –
+  and Susy needs new settings at each breakpoint.**
+  Susy3 is designed without mixins,
+  for complete flexibility,
+  but it can be useful to build additional tools
+  and shortcuts as we go.
+  Here are some snippets to help you get started
+  working with Susy across different media-queries.
+
+
+Susy3 Media-Queries & Grid Settings
+===================================
+
+Since Sass is not aware of the browser or DOM,
+there is no way to change Susy settings in the browser dynamically –
+where the adjusted variables would apply automatically
+at screen-widths.
+We'd have to use `CSS variables to achieve that`_.
+
+.. _CSS variables to achieve that: https://codepen.io/mirisuzanne/pen/PboVrw
+
+.. code:: css
+
+  /* ideal setup, using CSS variables */
+  @media (min-width: 30em) {
+    /* the new settings will apply to all elements */
+    /* above a viewport width of 30em */
+    :root { --susy-columns: 5; }
+  }
+
+In Sass
+we have to apply the *results* of our variable-changes explicitly,
+anywhere we want them used.
+Only the functions given new settings
+will generate CSS output based on those settings.
```

Comment:
I don't claim to be a Sass expert, but I'm having trouble following these two sentences.

Reply:
Does 0112808 help?